tnef.go

```diff
@@ -97,7 +105,7 @@ func DecodeFile(path string) (*Data, error) {
 // attachments and body into a Data object.
 func Decode(data []byte) (*Data, error) {
 	if byteToInt(data[0:4]) != tnefSignature {
-		return nil, errors.New("signature didn't match valid TNEF file")
+		return nil, ErrInvalidSignature
```

Comment:
Aside from the length-check changes (see my other comment), this one is probably a good one to add, since you will typically want to ignore these `ErrInvalidSignature` errors.

I could add that change for you if you want, or you can make a new PR with just this change.